README.md

```diff
@@ -1440,18 +1441,77 @@ to the same name. This raises an error:
 {% endcomponent %}
 ```
 
-## Passing data to components
```

Comment:
It's not clear from the diff, but I didn't remove this section, just renamed `Passing data to components` to `Accessing data passed to the component`, because that was the sole sub-section.